person/index_handler.go

```diff
@@ -47,19 +64,11 @@ func (i IndexHandler) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
 		resp.Results = append(resp.Results, *i.es.Index(p))
 	}
 
-	jsonResp, err := json.Marshal(resp)
-	if err != nil {
-		i.logger.Println(err.Error())
-		http.Error(rw, "Unable to encode response object to JSON", http.StatusBadRequest)
-		return
-	}
+	jsonResp, _ := json.Marshal(resp)
```

Comment:
Are we no longer interested in this error?

Reply:
It's one of those errors that can never be reached, unless the structure of `resp` (`IndexResponse`) is not marshalable which would be a persistent error picked up by automated tests.